examples/00-fluent/single_battery_cell_workflow.py

```diff
@@ -0,0 +1,719 @@
+            "uds-0": 1190000,
+            "uds-1": 983000,
+        },
+    },
+    {
+        "name": "p_material",
+        "chemical_formula": "pmat",
+        "density": 8978,
+        "specific_heat": 381,
+        "thermal_conductivity": 387.6,
+        "uds_diffusivity": {"option": "constant", "value": 10000000},
+    },
+    {
+        "name": "n_material",
+        "chemical_formula": "nmat",
+        "density": 8978,
+        "specific_heat": 381,
+        "thermal_conductivity": 387.6,
+    },
+]
+
+solids = solver.settings.setup.materials.solid
+for mat in materials:
+    solids.create(mat["name"])
+    solids[mat["name"]].chemical_formula = mat["chemical_formula"]
+    solids[mat["name"]].density.value = mat["density"]
+    solids[mat["name"]].specific_heat.value = mat["specific_heat"]
+    solids[mat["name"]].thermal_conductivity.value = mat["thermal_conductivity"]
+    if "uds_diffusivity" in mat:
+        solids[mat["name"]].uds_diffusivity = {
+            "option": mat["uds_diffusivity"]["option"]
+        }
+        if mat["uds_diffusivity"]["option"] == "defined-per-uds":
+            solids[mat["name"]].uds_diffusivity.uds_diffusivities["uds-0"].value = mat[
+                "uds_diffusivity"
+            ]["uds-0"]
+            solids[mat["name"]].uds_diffusivity.uds_diffusivities["uds-1"].value = mat[
+                "uds_diffusivity"
+            ]["uds-1"]
+        else:
+            solids[mat["name"]].uds_diffusivity.value = mat["uds_diffusivity"]["value"]
+
+#######################################################################################
+#  Defining the cell zone conditions
+# =====================================================================================
+# Assign e_material to e_zone → The active part of the battery cell (where the reactions happen)
+# is made of this material.
+#
+# Assign p_material to tab_pzone → The positive tab is made of the positive terminal material.
+#
+# Assign n_material to tab_nzone → The negative tab is made of the negative terminal material
+
+cell_zones = [
+    ("e_zone", "e_material"),
+    ("tab_nzone", "n_material"),
+    ("tab_pzone", "p_material"),
+]
+
+for zone, material in cell_zones:
+    solver.settings.setup.cell_zone_conditions.solid[zone].general.material = material
+
+#######################################################################################
+#  Defining Boundary Conditions
+# =====================================================================================
+# Convective heat transfer is applied to the battery’s external surfaces to model heat
+# loss to the environment, with a heat transfer coefficient of 5 W/m²·K:
+
+wall = solver.settings.setup.boundary_conditions.wall
+wall["wall_active"].thermal.thermal_condition = "Convection"
+wall["wall_active"].thermal.heat_transfer_coeff.value = 5
+
+solver.settings.setup.boundary_conditions.copy(
+    from_="wall_active", to=["wall_n", "wall_p"], verbosity=True
```

Comment:
Since it's unobvious what the implication of `verbosity=True` is, that should be commented here, or **preferably** the argument should be omitted.

Reason: We need to report the API design in this instance, as once again, **this does not belong in the API:** `verbosity:` `"Copy boundary conditions: Print more information."` in other words, the `verbosity=True` argument makes this method print directly to the console, which is undesirable for API calls shown in examples (or in general).